### DISCUSSION TITLE
How to adb connect to a factory reset android device

### DISCUSSION THREAD

**Discussion by Babydollll** [community/community#134455] [Programming Help]
Would be nice if you are the owner and have all the security info (my own real biometric data and my pin and pw.

**Comment by MLYTC1** [comment]
1. Enable Developer Options and USB Debugging:
After Factory Reset: When you first set up your device after a factory reset, you need to enable Developer Options and USB Debugging.
Access Developer Options:
Go to Settings.
Navigate to About Phone (or System > About Phone).
Tap on Build Number multiple times until you see a message saying "You are now a developer."
Enable USB Debugging:
Go back to Settings.
Navigate to Developer Options.
Enable USB Debugging.
2. Connect the Device via USB:
Use a USB cable to connect the device to your computer.
3. Check Device Connectivity:
Open a terminal or command prompt on your computer.
Run the command: adb devices
You should see a list of connected devices. If it's the first time connecting this device, you might need to authorize your computer on the device.
4. Authorize the Computer (if necessary):
When you connect the device for the first time with USB Debugging enabled, you should see a prompt on your Android device asking you to authorize the connected computer.
Tap Allow to grant permission.
5. Verify Connection:
Run adb devices again to ensure your device is listed. You should see something like:

arduino
Copy code
List of devices attached
1234567890abcdef    device
Additional Tips:
ADB Drivers: Ensure you have the necessary ADB drivers installed on your computer. For Windows, you might need to install specific drivers for your device model.
Device Authorization Issues: If you don’t see the authorization prompt, try reconnecting the USB cable, or reboot both the device and the computer.
USB Mode: Make sure your device is set to “File Transfer” (MTP) mode, which sometimes helps with connectivity issues.

**Comment by hubertit** [comment]
### Here is a clear step by step

1. **Enable Developer Options**:
   - Go to `Settings > About phone`.
   - Tap "Build number" seven times.
   - Go to `Settings > Developer options` and enable "USB debugging."

2. **Connect Device**:
   - Use a USB cable to connect the device to your computer.

3. **Check Device**:
   - Open a command prompt or terminal and type:
     ```bash
     adb devices
     ```
   - Ensure the device is recognized. Allow USB debugging on the device if prompted.

4. **Connect**:
   - Use `adb connect <device_ip>:<port>` if connecting over Wi-Fi, or use `adb shell` for a direct connection via USB.
